Pull documentation of ansible.module_utils from its doc strings. (#48416)
ansible/ansible: docs/docsite/rst/reference_appendices/module_utils.rst

Ansible Reference: Module Utilities
***************************************************************

This page documents utilities intended to be helpful when writing
Ansible modules in Python.

AnsibleModule
--------------

> This functionality is available via ``from ansible.module_utils.basic import AnsibleModule``.

Reviewer: ```suggestion
To use this functionality, include ``from ansible.module_utils.basic import AnsibleModule`` in your module.
```

Author: For the record: I had this in two places, fixed in both as of 48e8c74.

.. autoclass:: ansible.module_utils.basic.AnsibleModule
   :members:

Basic
------

This functionality is available via ``import ansible.module_utils.basic``:

.. automodule:: ansible.module_utils.basic

>    :members:

Reviewer 2: What does `members` limit this to?

Author: "Limit" does not describe it very well... Just saying `automodule::` just gives the doc string of the module itself. I considered it useful to also have the documentation of the module's members, so I added `:members:`. The [sphinx doc](http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) explains this.
